sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventsFilterForm.java

```diff
@@ -592,11 +592,11 @@ private void updateResponsibleUserFieldItems() {
 	private void updateResponsibleUserFieldItems(DistrictReferenceDto district, RegionReferenceDto region) {
 		final List<UserReferenceDto> items = new ArrayList<>();
 		if (district != null) {
-			items.addAll(FacadeProvider.getUserFacade().getUserRefsByDistrict(district, false, UserRole.SURVEILLANCE_OFFICER));
-			items.addAll(FacadeProvider.getUserFacade().getUsersByRegionAndRoles(region, UserRole.SURVEILLANCE_SUPERVISOR));
+			items.addAll(FacadeProvider.getUserFacade().getUserRefsByDistrict(district, false, null, UserRole.SURVEILLANCE_OFFICER));
+			items.addAll(FacadeProvider.getUserFacade().getUsersByRegionAndRoles(region, null, UserRole.SURVEILLANCE_SUPERVISOR));
```

Comment:
how ResponsibleUser filter will work on events since we know that on an event the disease is not mandatory